### PR TITLE
UF-8571: Added channel to API

### PR DESCRIPTION
**Pull request by foobar-1337**

### CODE REVIEW

File: src/main/java/se/sundsvall/supportmanagement/api/model/errand/Errand.java [hunk 1]
```diff
@@ -89,31 +92,31 @@ public class Errand {
...
 
 	@Schema(description = "Suspension information")
 	@Valid
 	private Suspension suspension;
 
 	@Schema(description = "Flag to indicate if the errand is business related", example = "true")
-	//TEMPORARY FIX: This should be set to true in version 6.0
-	//@NotNull(groups = {OnCreate.class, OnUpdate.class})
+	// TEMPORARY FIX: This should be set to true in version 6.0
+	// @NotNull(groups = {OnCreate.class, OnUpdate.class})
```
Comment (cheezi0747) [inline src/main/java/se/sundsvall/supportmanagement/api/model/errand/Errand.java:104]: Not related to this PR really but as this goes to 6.0 this should be activated.

File: src/main/java/se/sundsvall/supportmanagement/api/model/errand/Errand.java [hunk 2]
```diff
@@ -23,18 +25,15 @@
...
 
 	@Schema(description = "Unique id for the errand", example = "f0882f1d-06bc-47fd-b017-1d8307f5ce95", accessMode = READ_ONLY)
-	@Null(groups = {OnCreate.class, OnUpdate.class})
+	@Null(groups = { OnCreate.class, OnUpdate.class })
 	private String id;
 
 	@Schema(description = "Unique number for the errand", example = "KC-23010001", accessMode = READ_ONLY)
-	@Null(groups = {OnCreate.class, OnUpdate.class})
+	@Null(groups = { OnCreate.class, OnUpdate.class })
 	private String errandNumber;
```
Comment (AndreSoderlund) [inline src/main/java/se/sundsvall/supportmanagement/api/model/errand/Errand.java:37]: Might not be in scope of this story, but validating something this is READ_ONLY seems a bit excessive.

Comment (foobar-1337) [reply src/main/java/se/sundsvall/supportmanagement/api/model/errand/Errand.java:37]: It is "read_only" in the API, but this is not enforced (it's only an documentation-thingy). 
You can still send the errandNumber in the request, if you want to. 

I guess that's the idea behind this validation?

File: src/main/java/se/sundsvall/supportmanagement/api/model/errand/Errand.java (hunk 1 above)
Comment (foobar-1337) [reply src/main/java/se/sundsvall/supportmanagement/api/model/errand/Errand.java]: Ok, "activated" now.

**Pull request merged**